python/ray/_private/worker.py

```diff
@@ -1204,8 +1194,18 @@ def init(
         Exception: An exception is raised if an inappropriate combination of
             arguments is passed in.
     """
-    if configure_logging:
-        setup_logger(logging_level, logging_format or ray_constants.LOGGER_FORMAT)
+    deprecated_kwargs = ["configure_logging", "logging_level", "logging_format"]
```

Comment:
I don't think we should depreacte this. This is still needed to configure component-level loggers (e.g., dashboard_agent.py, dashboard.py, etc.).

Reply:
See discussion below. As per our conversation I've removed this for now, but let's submit an API change request so that we don't have this in the code base - users can already change library logger configuration using existing python logging methods.